src/test/java/tic/tac/toe/GameTest.java

```diff
@@ -0,0 +1,19 @@
+package tic.tac.toe;
+
+import org.junit.Test;
+import static org.junit.Assert.*;
+
+public class GameTest {
+  @Test
+  public void testGameHasAGreeting() {
+    Game game = new Game();
+    assertNotNull("game should have a greeting", game.getGreeting());
+  }
+  
+  @Test
+  public void testNewGameBoardIsEmpty() {
```

Comment:
It's good practice to have test names describing the behavior of the code vs. any structure of the content. i.e. `NewGameDisplaysEmptyBoard`